app/models/project.rb

```diff
@@ -310,7 +310,7 @@ def recurring?
   end
 
   def color
-    recurring? ? CatarseSettings[:default_color] : category.color
+    @color ||= channel_color || category_color || CatarseSettings[:default_color]
```

Comment:
Another question: Do you guys use decorators in the project?
If yes, could this method be moved to the decorator?